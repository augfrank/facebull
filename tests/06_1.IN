M0   C0   C3  2991
M1   C1   C5  2465
M2   C2   C1  2198
M3   C3   C1  2889
M4   C3   C2  2432
M5   C3   C4  2624
M6   C4   C0  2653
M7   C5   C4  2392
