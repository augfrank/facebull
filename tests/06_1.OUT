15131
0 1 2 4 6 7
